packages/eslint-plugin/jest.config.ts

```diff
@@ -5,7 +5,7 @@ export default {
   },
   moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'html'],
   globals: {},
-  displayName: 'eslint-plugin-nx',
+  displayName: 'packages-eslint-plugin',
```

Comment:
non-blocking: Does this need the `packages-` prefix? We don't use that elsewhere for this